api/google-meets/documentation/1.0.0/google-meets.json

```diff
@@ -404,6 +402,15 @@
               "newMeetingTimes": {
                 "type": "component"
               },
+              "opportunities": {
```

Comment:
Is this replacing openPositions?

Reply:
Yes

Reply:
Sounds like this should store a reference to the opportunities table.

Reply:
Yeah I've created a relationship between projects and opportunities collection. And once we have the ideas platform I expect them to do the same with ideas collection

Reply:
I think it was many to many relationship